engine/lib/output.php

```diff
@@ -468,6 +468,67 @@ function elgg_strip_tags($string, $allowable_tags = null) {
 }
 
 /**
+ * Convert an Elgg icon class to a corresponding FontAwesome icon
+ * for BC reasons
+ *
+ * @param string $elgg_icon_class the Elgg icon class name
+ *
+ * @see elgg_view_icon()
+ * @since 2.0
+ *
+ * @return string
+ */
+function elgg_convert_elgg_icon_to_fontawesome_icon($elgg_icon_class) {
```

Comment:
probably not but i don't know the policy for calling private functions in (public) views